packages/python-uncompyle6/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname=python-uncompyle6
+pkgver=1996.616e5c8
+pkgrel=1
+pkgdesc='A Python cross-version decompiler.'
+groups=('blackarch' 'blackarch-decompiler')
+arch=('any')
+url='https://github.com/rocky/python-uncompyle6'
+license=('GPL3')
+depends=('python2' 'python2-hypothesis' 'python2-pytest')
+makedepends=('git' 'python2')
+source=('git+https://github.com/rocky/python-uncompyle6.git')
+sha512sums=('SKIP')
+
+pkgver() {
+  cd python-uncompyle6
+  
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```

Comment:
Always add the last to the `$pkgver` if one is existent.

Reply:
Unfortunately, any more it isn't necessary.